configure.ac

```diff
@@ -253,6 +254,13 @@ if [test "$CUDA_H_PATH" ]; then
 	LIBS=$LIBS" -lcuda"
 fi
 
+if [test "$NEURON_PATH" ]; then
```

Comment:
Can we do this the proper way and introduce a configure flag with AC_ARG_WITH instead? The existing `CUDA_H_PATH` method in itself is not clean, we shouldn't add more such options.

Reply:
FWIW, I have a PR to fix this for CUDA https://github.com/linux-rdma/perftest/pull/152

Reply:
Thanks, changed to using AC_ARG_WITH as in your PR.

Reply:
The autoconfig updates look good to me.